downstream/modules/platform/proc-install-rh-sso-operator.adoc

```diff
@@ -0,0 +1,17 @@
+[id="proc-install-rh-sso-operator_{context}"]
+
+= Installing the Red Hat SSO Operator
```

Comment:
Should we document this procedure? It's "third party" content and not specific to AAP docs. Adding this procedure means we have to maintain SSO content in our doc set.

I'd just add a pre-req to whatever procedure and point to the SSO docs: https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.6/html/server_installation_and_configuration_guide/operator#doc-wrapper